images/Dockerfile.ubuntu

```diff
@@ -34,6 +34,7 @@ RUN apt install -y \
     jq \
     neovim \
     open-vm-tools \
+    snapd \
```

Comment:
I've checked, and this adds around ~30MB, I think that's a good compromise and add it in the image - no small libs, just snapd gets in